css-view-transitions-1/Overview.bs

```diff
@@ -61,82 +86,346 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
+	This extra in-between state often results in UX and accessibility issues,
+	as the structure of the DOM is compromised for a purely-visual effect.
+
+	View transitions avoid this troublesome in-between state by allowing the DOM to switch between states synchronously,
+	then performing a customizable visual transition between the two states in another layer,
+	using a static capture of the old state, and a live capture of the new state.
+
+	These captures are used in a tree of pseudo-elements
+	(detailed in [[#view-transition-pseudos]]),
+	where the old state cross-fades with the new state,
+	while animating from the old to new size and position.
+
+	The developer chooses which elements are captured independently,
+	meaning they can be animated independently,
+	using the 'view-transition-name' CSS property.
+
+## Lifecycle ## {#lifecycle}
+
+	A successful view transition goes through the following phases:
+
+	1. Developer calls <code>document.{{Document/startViewTransition}}({{UpdateCallback|updateCallback}})</code>,
+		which returns <code>viewTransition</code>, a {{ViewTransition}}.
+
+	1. Current state captured as the "old" state.
+
+	1. Rendering paused.
+
+	1. Developer's {{UpdateCallback|updateCallback}} is called,
+		where they update document state.
+
+	1. <code>viewTransition.{{ViewTransition/updateCallbackDone}}</code> fulfills.
+
+	1. Current state captured as the "new" state.
+
+	1. Transition pseudo-elements created.
+		See [[#view-transition-pseudos]] for an overview of this structure.
+
+	1. Rendering unpaused, revealing the transition pseudo-elements.
+
+	1. <code>viewTransition.{{ViewTransition/ready}}</code> fulfills.
+
+	1. Pseudo-elements animate until finished.
+
+	1. Transition pseudo-elements removed.
+
+	1. <code>viewTransition.{{ViewTransition/finished}}</code> fulfills.
+
+	<div id="phases-diagram">
+		<spec-scaler canvaswidth="1920" canvasheight="1080" style="aspect-ratio: 1920/1080">
+			<iframe class="spec-slides" src="diagrams/phases/"></iframe>
+		</spec-scaler>
+		<p class="spec-slide-controls">
+			<button disabled>Previous</button>
+			<button disabled>Next</button>
+		</p>
+		<script type="module">
+			const root = document.querySelector('#phases-diagram');
+			const [previous, next] = root.querySelectorAll('.spec-slide-controls button');
+			const iframe = root.querySelector('iframe');
+			next.disabled = false;
+
+			const updateButtons = (slide) => {
+				next.disabled = !slide.hasNext;
+				previous.disabled = !slide.hasPrevious;
+			};
+
+			next.addEventListener('click', async () => {
+				const slide = iframe.contentDocument.querySelector('spec-slide');
+				await slide.next();
+				updateButtons(slide);
+			});
+			previous.addEventListener('click', async () => {
+				const slide = iframe.contentDocument.querySelector('spec-slide');
+				await slide.previous();
+				updateButtons(slide);
+			});
+		</script>
+	</div>
+
+## Transitions as an enhancement ## {#transitions-as-enhancements}
```

Comment:
We already had a section titled this, but I felt it went a bit beyond describing how transitions are just an enhancement. It had lots of good stuff around the structure of the pseudos, so I moved that into the section about the pseudos.